client/src/components/Footer/Footer.tsx

```diff
@@ -23,7 +22,7 @@ const Footer: FC<FooterProps> = ({ versionNumber }) => {
           >
             <img src={getStaticPath(logo as string)} alt="inferno logo" className={styles.logo} />
           </Link>
-          <Typography>{version}</Typography>
+          <Typography className={styles.version}>{'v ' + version}</Typography>
```

Comment:
I'm not sure if it makes a huge difference, especially if it is so simple like this, but I think string interpolation instead of string concatenation is a good practice now that javascript supports it.

Reply:
Took me about 10 minutes to figure out it uses `` .... which are not the same as ''